src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -297,7 +297,7 @@ public function has_element_in_button_scope( $tag_name ) {
 	 * @param string $tag_name Name of tag to check.
 	 * @return bool Whether given element is in scope.
 	 */
-	public function has_element_in_table_scope( $tag_name ) {
+	public function has_element_in_table_scope( string $tag_name ): bool {
 		throw new WP_HTML_Unsupported_Exception( 'Cannot process elements depending on table scope.' );
```

Comment:
If this were reached it would be a type error because that constructor expects 6 arguments. It's expected to be used in a parsing context where we have information about tokens and parsing.

https://github.com/WordPress/wordpress-develop/blob/ca7d79505cbfe3c103e5ef96487e2b00e3a3244e/src/wp-includes/html-api/class-wp-html-unsupported-exception.php#L93-L105

Reply:
let's let this slide since we're closing up table scope in #6040